src/content/developers/docs/gas/index.md

```diff
@@ -46,13 +46,23 @@ Although a transaction includes a limit, any gas not used in a transaction is re
 ![Diagram showing how unused gas is refunded](../transactions/gas-tx.png)
 _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
 
+## What is gas limit? {#what-is-gas-limit}
+
+Gas limit refers to the maximum amount of gas you are willing to consume on a transaction. A higher gas limit means more computational work can be done while interacting with [smart contracts](/developers/docs/smart-contracts/). A standard ETH transfer requires a gas limit of 21,000 units of gas.
```

Comment:
I think this paragraph is a bit confusing – specifically the second sentence... 

Try:

More complicated transactions, involving smart contracts, require more computational work so have a higher gas limit than a simple payment. A standard ETH transfer...